src/js/app.js

```diff
@@ -2,17 +2,13 @@
     .controller('MainController', function () {
       this.stringMethods = Object.keys(STRING_EXTENSION);
       this.response = '';
-      this.evaluate = function (inputString) {
-        const stringMethod = document.getElementById('stringMethod').value;
-        const display = document.getElementById('display');
-        let currentFunc = '';
+      this.evaluate = function () {
```

Comment:
### Nitpick

You should use big arrow functions for anonymous functions instead.